Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Beckn/ACL/Common/Order.hs

```diff
@@ -235,7 +235,7 @@ tfAssignedReqToOrder Common.DRideAssignedReq {..} mbFarePolicy becknConfig = do
       arrivalTimeTagGroup = if isValueAddNP then Utils.mkArrivalTimeTagGroupV2 ride.driverArrivalTime else Nothing
       quote = Utils.tfQuotation booking
       farePolicy = FarePolicyD.fullFarePolicyToFarePolicy <$> mbFarePolicy
-      items = Utils.tfItems booking merchant.shortId.getShortId Nothing farePolicy
+      items = Utils.tfItems booking merchant.shortId.getShortId Nothing farePolicy Nothing
```

Comment:
why is payment.id linked to item here is Nothing? We do have payment object here

Reply:
Will fix this in my branch